src/com/palmergames/bukkit/towny/event/PlayerEnterTownEvent.java

```diff
@@ -3,13 +3,14 @@
 import com.palmergames.bukkit.towny.object.Town;
 import com.palmergames.bukkit.towny.object.WorldCoord;
 import org.bukkit.entity.Player;
+import org.bukkit.event.Cancellable;
 import org.bukkit.event.Event;
 import org.bukkit.event.HandlerList;
 import org.bukkit.event.player.PlayerMoveEvent;
 
-public class PlayerEnterTownEvent extends Event {
+public class PlayerEnterTownEvent extends Event implements Cancellable{
```

Comment:
How are we supposed to cancel a player walking into town? It was decided that stopping players from physically entering towns would not be done due to the strain it would put on servers.